src/utils/smartCamelize.ts

```diff
@@ -0,0 +1,30 @@
+import camelCase from 'lodash.camelcase';
+
```

Comment:
We should not be importing this directly as we already have it from frontend-platform in `camelizeObject`.  In general, I think the logic to not camelize some fields should just be explicit in the consumers, not in a general-use tool like this.